www/squid/src/opnsense/service/templates/OPNsense/Syslog/local/squid_access.conf

```diff
@@ -2,5 +2,5 @@
 # Local syslog-ng configuration filter definition [squid_access].
 ###################################################################
 filter f_local_squid_access {
-    program("^(squid-|squid-coord-)\\d+");
+    program("squid-*");
```

Comment:
So this globs instead of regex? Or are you missing a “.” now before “*”?

Reply:
@fichtner Yes in syslog's default it seems to only accept glob syntax. Type regex has to be specified explicitly. 
This one works as expected:
```
filter f_local_squid_access {
    program("squid-*");
};
```

Reply:
Ok just checking 😊